### ISSUE TITLE
Can we bring back the feature where we can use .env variable in Email Notifications?

### ISSUE THREAD

**Issue by 0xElectric** [verbb/formie#1695]
### What are you trying to do?

I used to be able to use .env variables e.g. $EMAIL_1 in 'Recipient Emails' in 'Email Notifications', now it just throws error:
```
Failed to send notification email: {"error":"Notification email parse error for “To: $EMAIL_1. Template error: “Email \"$email_1\" does not comply with addr-spec of RFC 2822.” \/var\/www\/html\/vendor\/symfony\/mime\/Address.php:54"}.
```

Why was it removed?

### What's your proposed solution?

...

**Comment by engram-design** [comment]
Sorry about that, looks to be due to a fix where we enforced lowercase emails, as some mailers didn't like non-lowercase emails (despite being pretty okay).

Should be fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.2"`.

**Comment by engram-design** [comment]
Fixed in [2.1.3](https://github.com/verbb/formie/releases/tag/2.1.3)
